src/UI/MvePlayer.cpp

```diff
@@ -908,9 +890,9 @@ void MvePlayer::think()
 {
     if (!_timerStarted) return;
 
-    struct timespec ts;
-    clock_gettime(CLOCK_MONOTONIC, &ts);
-    uint32_t nsec = (ts.tv_nsec - _lastts.tv_nsec);
+    struct TimeInfo ts;
```

Comment:
You don't need to write `struct SomeName var;` in C++.
Also you should consider returning of this structure by value.
Sometimes it can be more efficient than passing a pointer and writing to it.